swap_meet/vendor.py

```diff
@@ -1,2 +1,74 @@
+from .item import Item
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
```

Comment:
Good work in handling default parameters for mutable types. It's unsafe to set [] as the default value, but we can use None to stand in for the value when not supplied by the caller, and then use a new empty list in its place.

Reply:
We can also use a ternary to write this logic in one line:

```suggestion
    def __init__(self, inventory=None):
            self.inventory = inventory if inventory else []

```